package/src/module.js

```diff
@@ -91,6 +91,15 @@ export const makeDeserializer = (jsx) => {
         const attrs = ELEMENT_TAGS[nodeName](el)
         return jsx('element', attrs, children)
       }
+      if (
+        el.attributes &&
+        el.attributes.getNamedItem('class') &&
+        el.attributes
+          .getNamedItem('class')
+          .value.match(/msocomanchor|MsoCommentText/g)
+      ) {
+        return null  // word comment anchor and text annotations
+      }
```

Comment:
IDK if you think this should be the actual spot for this code, maybe there is a better place. But this removes the anchor link for the comment, and the comment body as well from pasting it in. 

Also I didn't write any tests for this, I was trying to figure out how to get a whole block of html, but didn't have the time.